source/core/slang-uint-set.h

```diff
@@ -19,23 +19,23 @@ constexpr Index intLog2(unsigned x)
     return x == 1 ? 0 : 1 + intLog2(x >> 1);
 }
 
+// if `in` is 0, result is undefined behavior
 static inline Index bitscanForward(uint64_t in)
 {
+    assert(in != 0);
 #if defined(_MSC_VER)
 
 #ifdef _WIN64
     uint64_t out = 0;
     _BitScanForward64((unsigned long*)&out, in);
     return Index(out);
```

Comment:
I saw that one of your code change wrapped __BitScanForward64 with `assert`.
Anything inside of `assert` will be removed in Release build so don't do it.
If you must, you can use SLANG_RELEASE_ASSERT`.

Reply:
I made a mistake when I did `assert()` with a function that has a 'out' parameter, I forgotten the function had a 'out' parameter when I made the change, this is why I adjusted the assert accordingly in the commit after.

I did not know Slang had a `SLANG_RELEASE_ASSERT`, thank you for bringing it up.